templates/vue/src/components/Lightbox/media/VideoMedia/H5PVideoMedia/H5PIframe.vue

```diff
@@ -6,9 +6,8 @@
       'context-multi-content': hasMultiContentContext,
     }"
     :style="{
-      height: frameHeight ? frameHeight + 'px' : 'auto',
-      width: frameWidth ? frameWidth + 'px' : '100%',
-      opacity: loading ? 0 : 1,
+      height: `${dimensions.height}px`,
+      width: '100%',
```

Comment:
This is probably what's causing the videos to be full width, even when they shouldn't be. I found that even if I set this to `${dimensions.width}px`, the iframe inside will still be cut off, so you might need to figure out when the width is off and refresh the iframe so it fits in properly.

Reply:
This is now open as a separate PR - 1059